src/data/primary-options.js

```diff
@@ -19,11 +19,5 @@ const options = [
 ];
 
 export const primaryOptions = options.sort((x, y) => {
-  if (x.value < y.value) {
-    return -1;
-  }
-  if (x.value > y.value) {
-    return 1;
-  }
-  return 0;
+   return x.value > y.value ? 1 : x.value===y.value ? 0 : -1;
```

Comment:
What's your thought on if we order using the `label` key vs the `value` key since this is what users will see. Also what if we use the `lodash.sortBy` function that can handle complex object sorting?

```suggestion
/** Imports */
import sortBy from 'lodash.sortby';

/** Exports */
export const primaryOptions = sortBy(options, ['label']);
```

Reply:
I believe the current sortBy function works, I am not sure if a change (and an import of the lodash library) is really so necessary for something that isn't broken. The really concise ternary operator solution wasn't merged for readability sake. It was very terse almost to the point of forsaking readability.